src/ingest_validation_tools/directory-schemas/codex.yaml

```diff
@@ -17,50 +10,62 @@
   description: 'PDF export of Powerpoint slide deck containing the Image Analysis Report'
   is_qa_qc: True
 -
-  pattern: 'drv_[^/]+/channelNames\.txt'
-  description: 'Text file produced by the Akoya software which contains the (linearized) channel number and the Name/ID/Target of the channel (required for HuBMAP pipeline)'
+  pattern: '(raw|src_.*)/.*'
+  description: 'Raw data directory.'
   required: True
 -
-  pattern: 'src_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(processed|drv_.*)/.*'
+  description: 'Processed files produced by the Akoya software or alternative software.'
   required: True
 -
-  pattern: 'drv_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(raw|src_.*)+exposure_times\.txt'
```

Comment:
Maybe...
```suggestion
  pattern: '(raw|src_[^/]*)/exposure_times\.txt'
```
`+` allows repetition of the previous. `rawrawrawexposure_times.txt` is probably not what you meant.

Reply:
@mccalluc I am not getting the same result as you suggested. Am I using the wrong flavor of regex? This is using the [`re`](https://github.com/python/cpython/blob/3.9/Lib/re.py).
 
![Screen Shot 2021-07-21 at 5 01 04 PM](https://user-images.githubusercontent.com/413752/126559553-d89f6b2b-bf65-4c8d-bfce-7955d1765432.png)

I'll accept your suggestion, I am just trying to figure out what am I doing wrong.